python/mcap/tests/test_writer.py

```diff
@@ -17,7 +17,7 @@
 def generate_sample_data(compression: CompressionType):
     file = TemporaryFile("w+b")
     writer = Writer(file, compression=compression)
-    writer.start(profile="x-json", library="test")
+    writer.start(profile="", library="test")
```

Comment:
Ideally `profile` could be an optional argument?

Reply:
+1

Reply:
It already is.

Reply:
I see this is a test not an example, so I don't feel as strongly but it would be nice to drop it entirely from here rather than use an empty string. Or leave `x-json` since again this is a test.